packages/rich-text/src/plugins/Quote/utils.ts

```diff
@@ -7,51 +7,53 @@ import { CustomElement } from '../../types';
 
 const LIST_TYPES: BLOCKS[] = [BLOCKS.OL_LIST, BLOCKS.UL_LIST];
 
-export function toggleBlock(editor, type: string): void {
+export function toggleBlock(editor: PlateEditor, type: string): void {
   const isActive = isBlockSelected(editor, type);
   const isList = LIST_TYPES.includes(type as BLOCKS);
   const isQuote = type === BLOCKS.QUOTE;
 
-  Transforms.unwrapNodes(editor, {
-    match: (node) => {
-      if (Editor.isEditor(node) || !Element.isElement(node)) {
-        return false;
-      }
-
-      // Lists
-      if (isList && LIST_TYPES.includes((node as CustomElement).type as BLOCKS)) {
-        return true;
-      }
+  Editor.withoutNormalizing(editor, () => {
```

Comment:
This change is not relevant to this PR specifically, I just noticed it when trying to figure out the issue and thought it makes sense.